Tribler/Core/Category/category.conf

```diff
@@ -12,7 +12,7 @@ matchpercentage = 0.5
 
 [VideoClips]
 rank = 2
-displayname = Video Clips
+displayname = VideoClips
```

Comment:
Why not display it with the proper spelling (which is with a space between the words)?

Reply:
This thing is only used to tag Legacy channels now. However, the tag should be a single word. Hence, this change.